distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -2532,9 +2531,13 @@ public void testInjectAttributesOnCreate() throws Exception {
                     .assertThat()
                     .body(hasXPath(basicMetacardXpath + "/type", is(basicMetacardTypeName)))
                     .body(hasXPath(basicMetacardXpath + "/int[@name='page-count']/value", is("55")))
+                    .body(hasXPath(basicMetacardXpath + "/string[@name='point-of-contact']"),
+                            containsString("Guest@Guest"))
                     .body(not(hasXPath(basicMetacardXpath + "/double[@name='temperature']")))
                     .body(hasXPath(otherMetacardXpath + "/type", is(otherMetacardTypeName)))
                     .body(hasXPath(otherMetacardXpath + "/int[@name='page-count']/value", is("55")))
+                    .body(hasXPath(otherMetacardXpath + "/string[@name='point-of-contact']"),
+                            containsString("Guest@Guest"))
```

Comment:
Yes. It's the core use case. PoC is most valuable when someone is logged in. And we want to catch a change in Shiro that would impact marking.